mrbelvedereci/build/models.py

```diff
@@ -271,8 +273,19 @@ def get_project_config(self):
         project_config.set_keychain(keychain)
         return project_config
 
-    def get_org(self, project_config):
-        org_config = project_config.keychain.get_org(self.plan.org)
+    def get_org(self, project_config, retries=3):
+        def get_org_config(attempt=None):
```

Comment:
I think a while True loop would be more easy to quickly understand (like over here https://github.com/SalesforceFoundation/CumulusCI/blob/master/cumulusci/core/tasks.py#L148) but i won't say no to this because of that